tools/python/boutiques/localExec.py

```diff
@@ -109,11 +108,16 @@ def execute(self):
       # Change launch (working) directory if desired
       launchDir = '${PWD}' if (self.launchDir is None) else self.launchDir
       # Run it in docker
+      mount_strings = [] if not mount_strings else mount_strings
+      mount_strings.append('${PWD}:' + launchDir)
       if conType == 'docker':
-        dcmd = 'docker run --entrypoint=/bin/bash --rm' + envString + ' -v ${PWD}:' + launchDir + ' -w ' + launchDir + ' ' + str(conImage) + ' ./' + dsname
+        # export mounts to docker string
+        docker_mounts = " -v ".join(m for m in mount_strings)
+        dcmd = 'docker run --entrypoint=/bin/bash --rm' + envString + ' -v '+docker_mounts+ ' -w ' + launchDir + ' ' + str(conImage) + ' ./' + dsname
```

Comment:
Can you confirm if we need the `-v` on this line since we add it to the `docker_mounts` variable above? I never remember if the join prepends the separator as well. (Would test but on my phone, will test when back at a computer if you haven't replied yet)